src/main/java/org/stellar/base/AssetTypeNative.java

```diff
@@ -7,6 +7,16 @@
   public AssetTypeNative() {}
 
   @Override
+  public String getCode() {
```

Comment:
idk that doesn't seem too crazy of an assumption to me but I guess they are only used to compare if two Assets are the same right now. probably better to just override `public boolean equals(Object object) {` for each class that derives from `Asset`